drivers/pwm/pwm_nrfx.c

```diff
@@ -131,6 +197,13 @@ static int pwm_nrfx_init(struct device *dev)
 	(CONFIG_PWM_##dev_idx##_NRF_CH##ch_idx##_INVERTED ? \
 	 PWM_NRFX_CH_VALUE_INVERTED : PWM_NRFX_CH_VALUE_NORMAL)
 
+
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
+#define PM_FUNCTION(idx) pwm_##idx##_nrfx_pm_control
```

Comment:
You don't need this. Just use `pwm_##idx##_nrfx_pm_control` directly. When power management is not enabled, this parameter is cut out (see [DEVICE_DEFINE](https://github.com/zephyrproject-rtos/zephyr/blob/master/include/device.h#L135)).

Reply:
done